### ISSUE TITLE
Fix splash page mobile

### ISSUE THREAD

**Issue by karacampbell** [karacampbell/daaas#47]
![image](https://user-images.githubusercontent.com/41926868/83566124-7f68cb00-a4ed-11ea-81d4-5709f81296cb.png)

**Comment by TheWilks** [comment]
Saw this open just wanted to make a suggestion that we align it with the sign in pages. Thoughts?

**Comment by karalaina** [comment]
@TheWilks  Yeah I agree for sure. Do you mean the whole page or just the mobile view? I think aligning the whole page would be nice, but idk what to do with the image. Also, we talked a while back about changing this image, is that still the plan?

**Comment by haze0028** [comment]
I was messing with styles, came up with some options. It gets a bit squish on the iphone 5 view. We could just ignore the margins. Coping the sign-in layout could definitely work.

![2020-06-05 13_08_48-DAaaS](https://user-images.githubusercontent.com/16760818/83904607-2c318b00-a72e-11ea-8ba3-85b288630635.jpg)
![2020-06-05 13_10_05-DAaaS](https://user-images.githubusercontent.com/16760818/83904609-2cca2180-a72e-11ea-83ea-aaab97ce356c.jpg)
![2020-06-05 13_10_33-DAaaS](https://user-images.githubusercontent.com/16760818/83904610-2d62b800-a72e-11ea-984f-575d97456dcb.jpg)

**Comment by TheWilks** [comment]
My daughter is down for a nap now. I can throw something together quickly but I would still archive the current stuff. I’ll post here when done.

**Comment by TheWilks** [comment]
![Official language](https://user-images.githubusercontent.com/6516828/83913907-32c7fe80-a73e-11ea-9c78-a80f862da870.png)

**Comment by TheWilks** [comment]
![XS - Official language](https://user-images.githubusercontent.com/6516828/83913922-3c516680-a73e-11ea-8f5e-0fd6ce47440e.png)

**Comment by TheWilks** [comment]
Just getting a strange bug where the line is under the buttons present sometimes. Have not tested all screen sizes.

![A95B7260-BC12-4526-9DCA-BDC1AA57B197](https://user-images.githubusercontent.com/6516828/84087049-4c27af80-a9b7-11ea-92c0-db367a6ca284.png)

**Comment by TheWilks** [comment]
Just getting a strange bug where the line is under the buttons present sometimes. Have not tested all screen sizes. 

It seems to happen when I rotate my phone. This may be an underlying issue with something else for example things zoom in on rotate. 

Try on your mobile as well to narrow down if it’s just an iPhone issues.

The line...

![A95B7260-BC12-4526-9DCA-BDC1AA57B197](https://user-images.githubusercontent.com/6516828/84087049-4c27af80-a9b7-11ea-92c0-db367a6ca284.png)

The rotate zoom...

![0626652B-357D-4235-8D8B-F824C44BA5D2](https://user-images.githubusercontent.com/6516828/84087893-28656900-a9b9-11ea-83a9-dcd03c973ddc.png)

**Comment by haze0028** [comment]
Unforunately I can't get it to show up on my phone, or in the browser when I try different screen sizes in Chrome. Must be an iphone issue. I'll work on it today and see if I can figure out a solution
